packages/shared/components/MainMenu.svelte

```diff
@@ -1,78 +1,78 @@
 <script lang="typescript">
-    import { Drawer, Icon, Text } from 'shared/components'
-    import { logout } from 'shared/lib/app'
-    import { isBright, getInitials } from 'shared/lib/helpers'
+	import { Drawer, Icon, Text, ProfileActions } from 'shared/components'
+    import { getInitials } from 'shared/lib/helpers'
     import { activeProfile, getColor } from 'shared/lib/profile'
     import { selectedAccount } from 'shared/lib/wallet'
     import { Settings } from 'shared/routes'
-    import { settingsRoute, settingsRouter, SettingsRoute } from '@core/router'
-    import { Locale } from '@core/i18n'
+    import { 
+        profileRoute, 
+        profileRouter, 
+        ProfileRoute, 
+        settingsRoute, 
+        settingsRouter, 
+        SettingsRoute 
```

Comment:
Get linting errors because of trailing spaces

Reply:
🙏 done!